tests/GoogleAuthenticatorTest.php

```diff
@@ -34,9 +34,11 @@ public function testGenerateSecret()
     }
 
     /**
+     * @group legacy
+     * @expectedDeprecation Passing anything other than null or a DateTimeInterface to $time is deprecated as of 2.0 and will not be possible as of 3.0.
```

Comment:
I checked the code, it does not look like there is an alternative indeed.

> it doesn't seem to crash the current setup though

Why would it?